stock_move_location/tests/test_common.py

```diff
@@ -118,12 +118,20 @@ def setup_product_amounts(self):
         )
 
     def set_product_amount(
-        self, product, location, amount, lot_id=None, package_id=None, owner_id=None
+        self, product, location, target_qty, lot_id=None, package_id=None, owner_id=None
```

Comment:
@joshkreud In fact, I wouldn't have changed the signature here as some would have inherited from it in further modules. Can you change ?

Reply:
Yeah I actually thought the same, but since the function previously did not and now does what you would expect from it's name, I thought it would be good to provoke an error in downstream tests. 
But I reset the signature and added a docstring clarifying what the function does now.